src/wolff_client.erl

```diff
@@ -37,10 +37,12 @@
 
 -type config() :: map().
 -type topic() :: kpro:topic().
+-type alias_and_topic() :: wolff_producers:alias_and_topic().
```

Comment:
the type should be defined in this module.
otherwise it's a loop dependency

Reply:
this seems to work just fine for dialyzer?

Reply:
yes, dialyzer is ok with it.
it's just a good practice to avoid cyclic ref, some static code analyzer may not work well.

Reply:
:checkered_flag: